localstack.yaml

```diff
@@ -0,0 +1,195 @@
+      find ${{targets.destdir}}/usr/share/${{package.name}}/.venv/lib/python${{vars.py-version}}/site-packages/ -type d -name "tests" -exec rm -rf {} +
+
+  - name: Install scripts
+    working-directory: bin
+    runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/bin
+      for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+        install -Dm755 $f ${{targets.contextdir}}/usr/bin/$f
+      done
+
+  - name: Mark community version
+    runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/lib/localstack
+      echo "${{package.version}} (Built by Chainguard)" > ${{targets.contextdir}}/usr/lib/localstack/.community-version
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream Dockerfile"
+    pipeline:
+      - name: Symlink scripts
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/bin
+          for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+            ln -sf /usr/bin/$f "${{targets.contextdir}}"/usr/local/bin/$f
+          done
+      - name: Symlink binaries and libraries
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/opt/code/localstack/bin
+          ln -sf /usr/share/localstack/.venv "${{targets.contextdir}}"/opt/code/localstack/.venv
+          for f in localstack localstack-supervisor; do
+            ln -sf /usr/bin/$f "${{targets.contextdir}}"/opt/code/localstack/bin/$f
+          done
+      - name: Symlink Java 11
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/var/lib/localstack/lib/java/11
+          for f in bin conf legal lib release; do
+            ln -sf /usr/lib/jvm/java-11-openjdk/$f "${{targets.contextdir}}"/var/lib/localstack/lib/java/11/$f
+          done
+      - name: Symlink Python ${{vars.py-version}}
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/lib
+          ln -sf /usr/lib/python${{vars.py-version}} "${{targets.contextdir}}"/usr/local/lib/python${{vars.py-version}}
+    test:
+      pipeline:
+        - name: tests for script symlinks
+          runs: |
+            for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+              stat /usr/local/bin/$f
+            done
+            for f in localstack localstack-supervisor; do
+              stat /opt/code/localstack/bin/$f
+            done
+            for f in bin conf legal lib release; do
+              stat /var/lib/localstack/lib/java/11/$f
+            done
+        - name: test symlinks
+          runs: |
+            stat /opt/code/localstack/.venv
+            stat /usr/local/lib/python${{vars.py-version}}
+            stat /usr/local/lib/python${{vars.py-version}}
+
+update:
+  enabled: true
+  github:
+    identifier: localstack/localstack
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - localstack-compat
+  pipeline:
```

Comment:
Just checking in, are we confident with tests? Anything out there to expand a bit? In the `post:` section, we can verify something with `curl` or smth for quick smoke testing. (i.e., health check, components status, plugins status, etc.)

Reply:
I found this in their Dockerfile health check: `.venv/bin/localstack status services --format=json`

Reply:
thank you for the reviews, I've addressed it via https://github.com/wolfi-dev/os/pull/39824/commits/6b8461dd79bfa59a9f6ab49e4e04aa8b25b46915 (added a separate test block that uses curl tests)